DOCKER.md

```diff
@@ -25,7 +25,13 @@ Find and open the Docker Desktop application. When you run your application, it
 
 Duplicate the `secrets.example.toml` file and rename the copy to `secrets.toml`. <br>Edit the `secrets.toml` file and change the `api_key` value under "default" to be your Vectara API key.
 
-Note: The use of `secrets.toml` allows you to create multiple profiles, and store a differnet API key under each profile. This may be helpful if you are working with multiple Vectara Answer apps in parallel. As you can see in the included `secrets.toml` file we've included the API keys (query only) for the existing datasets that are part of the Quickstart: Feynman, vectara.com and vectara docs.
+Note: The use of `secrets.toml` allows you to create multiple profiles, and store a differnet API key under each profile. This may be helpful if you are working with multiple Vectara Answer apps in parallel. As you can see in the included `secrets.toml` file we've included the API keys (query only) for the existing datasets that are part of the Quickstart: Feynman, vectara.com and vectara docs. Some secrets are shared among all profiles, and in order to avoid having to copy that secret under each profile, the code designates a special profile called "general" - each secret under this profile will be used in any profile (in addition to the profile-specific secrets). 
+
+Note: when computing the HEM (Hughes Hallucination Evaluation Model), the code calls the HEM huggingface space. If you sign up for a huggingface account you can include your [HF token](https://huggingface.co/docs/hub/security-tokens) under the "general" profile as follows:
```

Comment:
I think we should write out "Hugging Face" everywhere we refer to the service, the same as it's written on the Hugging Face website.

"huggingface" -> "Hugging Face"